app/common/data/interfaces/grants.py

```diff
@@ -13,13 +13,31 @@ def get_grant(grant_id: UUID) -> Grant:
     return db.session.get_one(Grant, grant_id)
 
 
+def grant_name_exists(name: str) -> bool:
+    statement = select(Grant).where(Grant.name == name)
+    grant = db.session.scalar(statement)
+    return grant is not None
+
+
 def get_all_grants() -> Sequence[Grant]:
     statement = select(Grant).order_by(Grant.name)
     return db.session.scalars(statement).all()
 
 
-def create_grant(name: str) -> Grant:
-    grant: Grant = Grant(name=name)
+def create_grant(
+    name: str,
+    description: str,
+    primary_contact_name: str,
+    primary_contact_email: str,
+    ggis_number: str | None = None,
+) -> Grant:
```

Comment:
```suggestion
def create_grant(
    *,
    name: str,
    description: str,
    primary_contact_name: str,
    primary_contact_email: str,
    ggis_number: str | None = None,
) -> Grant:
```

When there are more than 2-3 arguments for the interface, we've been tending to force them over to kwargs to make it more explicit that things are being passed into the right places.